pkg/apiserver/resthandler.go

```diff
@@ -567,11 +567,11 @@ func patchResource(ctx api.Context, admit updateAdmissionFunc, timeout time.Dura
 				return nil, err
 			}
 
-			currentPatch, err := strategicpatch.CreateStrategicMergePatch(originalObjJS, currentObjectJS, patcher.New())
+			currentPatch, err := strategicpatch.CreateStrategicMergePatch(originalObjJS, currentObjectJS, versionedObj)
```

Comment:
and already being passed to strategic patch computation at https://github.com/kubernetes/kubernetes/pull/23145/files#diff-fe6dbe5fa548ea5c031a5be1669a6376R531